packages/design-system/tailwind/colors.ts

```diff
@@ -1,4 +1,4 @@
-export const colors = {
```

Comment:
Isn't it the same or it should be `module.exports`, maybe because of compiling issues? Just curious 🤔

Reply:
Yeah the content is the same but we would like to import this file from the Tailwind config and we can't import an ES module from it